packages/flame_mini_sprite/lib/src/components/mini_sprite_component.dart

```diff
@@ -92,8 +99,8 @@ class MiniSpriteComponent extends PositionComponent {
         Rect.fromLTWH(
           entry.currentPosition.x * _pixelSize,
```

Comment:
How does this work, shouldn't it bleed in all directions? 🤔

Reply:
yeah, the way I implemented, it bleeds on all directions, initially I thought on identifying the pixels that required bleeding, but I ended up thinking that the overhead of that additional calculation it is not worth since the bleeding necessary is so little.

Reply:
Let me know if you think I should do differently @spydon

Reply:
But I'm this rect, doesn't it only cover bleeding to the right and downwards?

Reply:
Yes! I probably should add an offset too right?

Reply:
Yeah, that is exactly what I thought when mentioned the offset

Reply:
@spydon can you take another look, I did the same idea, but with a smaller "added margin"